### PR TITLE
fix(restore): set targetip on CVRs after restore is completed

### PR DESCRIPTION
**Pull request by zlymeda**
Auto-setting of targetip on CVR if openebs.io/restore-completed is found

## Pull Request template

**Why is this PR required? What issue does it fix?**:
Currently, users have to set targetip on CVRs after restore. Corresponding change in https://github.com/openebs/velero-plugin/pull/131 will mark CVRs which needs to update the targetip. Changes in this PR will check the annotation and set the targetip.

**What this PR does?**:
Sets targetip on CVRs marked with openebs.io/restore-completed annotation.

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
`make test` in velero-plugin repository with changes deployed.

**Any additional information for your reviewer?** : 
_Mention if this PR is part of any design or a continuation of previous PRs_

related velero-plugin PR: https://github.com/openebs/velero-plugin/pull/131
related cstor-operators PR: https://github.com/openebs/cstor-operators/pull/198

**Checklist:**
- [x] Fixes https://github.com/openebs/velero-plugin/issues/127
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Has the change log section been updated? 
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track: 
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: cmd/cstor-pool-mgmt/controller/replica-controller/handler.go [hunk 1]
```diff
@@ -281,6 +282,24 @@ func (c *CStorVolumeReplicaController) cVREventHandler(
 		if isCVRCreateStatus(cvrObj) {
 			return c.cVRAddEventHandler(cvrObj, fullVolName)
 		}
+
+		if cvrObj.Annotations[restoreCompletedAnnotation] == "true" {
+			targetIp, err := volumereplica.GetTargetIp(fullVolName)
+			if err != nil {
+				return "", err
```
Comment (mynktl) [inline cmd/cstor-pool-mgmt/controller/replica-controller/handler.go:289]: We need to log error here.
If returned status is empty then caller is ignoring the error. https://github.com/openebs/maya/blob/master/cmd/cstor-pool-mgmt/controller/replica-controller/handler.go#L134

**Review by mynktl — COMMENTED** [review]
Thanks for the PR @zlymeda .
PR looks good. Given one comment about logging the error. PTAL.

File: cmd/cstor-pool-mgmt/controller/replica-controller/handler.go (hunk 1 above)
Comment (mynktl) [reply cmd/cstor-pool-mgmt/controller/replica-controller/handler.go:289]: Also, logging this as an event on CVR will ease the debugging.

Comment (zlymeda) [reply cmd/cstor-pool-mgmt/controller/replica-controller/handler.go:301]: done, PTAL

**Review by mittachaitu — APPROVED** [review]
Changes are good

**Review by mynktl — APPROVED** [review]

**Comment by zlymeda** [comment]
I rebased with origin/master and squashed commits into single commit

**Pull request merged**